redisgraph/util.py

```diff
@@ -1,6 +1,8 @@
 import random
 import string
 
+__all__ = ['random_string', 'quote_string']
```

Comment:
Why is this necessary ?
when importing everything from this module (util.py) the importer will get `random_string` and `quote_string` by default.

Reply:
@swilly22 https://lgtm.com/projects/g/RedisGraph/redisgraph-py/alerts/?mode=list